pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -233,6 +233,9 @@ func (d *ProjectStatusDescriber) Describe(namespace, name string) (string, error
 			printLines(out, indent, 0, describeMonopod(f, monopod.Pod)...)
 		}
 
+		bcPipelines, _ := graphview.AllImagePipelinesFromBuildConfig(g, graphview.IntSet{})
```

Comment:
Yeah, `Images` got added to the `BuildSource` section of the `BuildConfig` as recently as late 2015 according to `git blame`.  It is a peer member to items like the Git connection information for example in the `BuildSource` struct, and separate from the "base image" specified in the `From` field of the 3 classic `BuildStrategy` types.  

It is a way to overlay files from additional images on top of any files produced by the "build" (overloaded term :-) ) performed by the s2i/docker/custom strategies.

There is a minor inspection of it in already `oc status` [here](https://github.com/openshift/origin/blob/master/pkg/cmd/cli/describe/projectstatus.go#L888-L890), but specific images are not mentioned.

With that background, next set of thoughts:
1) leave any expansion of the exposure of Images to the trello card opened based on @smarterclayton 's call for a formal proposal on pipelines, alignment with console, etc.
2) in the short term, I'll refactor this pull to generate the "produce by..." output noted earlier; through either further discussion here or as 1) progresses, if such a change is deemed as an acceptable, short-term/tactical change, we can move toward merge; as such, I'll change this PR to WIP in the interim